testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-ldap-and-file/password.db

```diff
@@ -0,0 +1 @@
+DefaultGroupUser:$2y$10$xA36wzOAnGWJmukr/zItyOrOyXPD6prgszOCN93MyFUpMAbGKcklm
```

Comment:
Please do not use same user here as in ldap. That way we don't know which authenticator was actually used.

Reply:
Part of the whole idea is to use two authenticators for one user.
I will another one and test it separately

Reply:
So how does it work then? You will one authenticator any way. It is not much interesting case.

Reply:
Multiple password authenticators works exactly the same as standard authenticators.